includes/CLI/Plugin_Check_Command.php

```diff
@@ -243,6 +243,12 @@ static function ( $dirs ) use ( $excluded_files ) {
 			$warnings = $result->get_warnings();
 		}
 
+		if ( empty( $errors ) && empty( $warnings ) ) {
+			WP_CLI::success( 'Checks complete. No errors found.' );
```

Comment:
I don't see the translation function...
__( '', 'plugin-check' )

Reply:
sorry, I hadn't noticed that other strings passed to `WP_CLI::line()`, etc were translated...I'm so used to strings in standard WP_CLI commands not being translated.

Another commit coming shortly.